codegenerator/cli/templates/static/codegen/src/eventFetching/ChainFetcher.res

```diff
@@ -47,9 +47,9 @@ let make = (
   ~processingFilters,
   ~maxAddrInPartition,
   ~dynamicContractPreRegistration,
+  ~enableRawEvents,
 ): t => {
   let module(ChainWorker) = chainConfig.chainWorker
-  logger->Logging.childInfo("Initializing ChainFetcher with " ++ ChainWorker.name ++ " worker")
```

Comment:
Removed the log since I couldn't find any value in it.

Reply:
I suppose the main value is to be able to see rpc vs hypersync for a given chain.